concurrent/src/test/java/io/airlift/concurrent/TestAsyncSemaphore.java

```diff
@@ -289,122 +288,135 @@ public void testNoStackOverflow()
         Futures.allAsList(futures).get(1, TimeUnit.MINUTES);
     }
 
-    @DataProvider
-    public static Object[][] concurrency()
+    public static int[] testedConcurrency()
```

Comment:
@ParameterizedTest

Reply:
We don't use it in trino as it's against the test guide. Sorry